spec/requests/api/v1/course_spec.rb

```diff
@@ -552,5 +494,61 @@ def get_next_courses(link, params = {})
         expect(Subject.where(subject_code: "U3").count).to eq(0)
       end
     end
+
+    describe "age_range" do
+      context "when level is 'primary'" do
+        let(:course) { create(:course, provider: provider, age_range_in_years: "3_to_7", level: "primary", subjects: [find_or_create(:primary_subject, :primary)]) }
+        let(:site) { create(:site_status, :published, course: course) }
+
+        before do
+          course
+          site
+
+          get "/api/v1/#{current_year}/courses",
+              headers: { "HTTP_AUTHORIZATION" => credentials }
+        end
+
+        it "should equal 'P'" do
+          json = JSON.parse(response.body).first
+          expect(json["age_range"]).to eq("P")
+        end
+      end
+
+      context "when age_range_in_years is '7_to_14'" do
+        let(:course) { create(:course, provider: provider, age_range_in_years: "7_to_14", level: "secondary", subjects: [find_or_create(:secondary_subject, :modern_languages)]) }
+        let(:site) { create(:site_status, :published, course: course) }
+
+        before do
+          course
+          site
+
+          get "/api/v1/#{current_year}/courses",
+              headers: { "HTTP_AUTHORIZATION" => credentials }
+        end
+
+        it "should equal 'M'" do
+          json = JSON.parse(response.body).first
+          expect(json["age_range"]).to eq("M")
+        end
+      end
+
+      context "default" do
```

Comment:
Minor point for the future: I think this could have a better name than "default", it's too tied to the implementation imo. Maybe it could be "secondary course" or something.